src/fix-links.test.js

```diff
@@ -9,15 +9,9 @@ describe('fixLinks', () => {
     test('should insert the <base> element into <head>', async () => {
         const doc = window.document.implementation.createHTMLDocument()
         const rootElement = doc.documentElement
-        doc.querySelector('head').insertAdjacentElement = jest.fn()
-        const docUrl = 'about:blank'
+        const docUrl = 'https://example.com/'
```

Comment:
Use e.g. `https://example.com/page`; it is nice to see whether the `docUrl` or the base of `docUrl` is used (should not be of importance in this test, but it feels like a good practice).